src/lib/geometry/socketsParts.ts

```diff
@@ -388,6 +388,22 @@ export const PART_INFO: Record<CuttleKey['type'], PartInfo> = {
     icon: 'knob',
     description: 'A horizontal knob that sits low to the keyboard. The part is no longer manufactured, so you will have to find a reseller online.',
   },
+  'encoder-alps-rkjxt1f42001': {
+    partName: 'Alps RKJXT1F42001',
+    bomName: 'Alps RKJXT1F42001',
+    category: 'Encoders',
+    stepFile: '/src/assets/encoder-alps-rkjxt1f42001.step',
+    socketSize: {
+      radius: 8.625,
+      height: 5,
+      sides: 20,
+    },
+    partBottom: [box(17.25, 17.25, 12)],
+    wiredInMatrix: true,
+    pinsNeeded: 7,
```

Comment:
I need to make a better system for indicating what goes into the matrix, because right now `wiredInMatrix` assumes there is only 1 button on the part. This one has 5 buttons, so I think the easiest way to resolve this would be to set `wiredInMatrix` to `false` and keep the 7 I/Os (5 buttons & 2 encoder pins). Common pins (GND) are not included in this count

Reply:
I made this change.